integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/cucumber-tests-platform-smartmetering.properties

```
@@ -54,3 +54,5 @@ jaxb2.marshaller.context.path.smartmetering.notification=org.opensmartgridplatfo
 web.service.smartmetering.notification.application.name=SMART_METERS
 web.service.smartmetering.notification.context=/notifications/
 web.service.smartmetering.notification.port=8189
+
+jre.encryption.key.resource=classpath:osgp-secret-management-jre.key
```

Consider using [same key as in secret-management](https://github.com/OSGP/open-smart-grid-platform/blob/development/osgp/platform/osgp-secret-management/src/main/resources/osgp-secret-management.properties#L3)
```suggestion
jre.encryption.key.resource=file:/etc/osp/smartmetering/keys/platform/osgp-secret-management-jre.key
```